new-log-viewer/src/typings/logs.ts

```diff
@@ -8,8 +14,20 @@ enum LOG_LEVEL {
     FATAL
 }
 
+type LogLevelFilter = Nullable<LOG_LEVEL[]>;
+
 const INVALID_TIMESTAMP_VALUE = 0;
 
+interface JsonLogEvent {
+    timestamp: Dayjs,
+    level: LOG_LEVEL,
+    fields: JsonObject
+}
```

Comment:
nit: let's try to place all types before values.
 
```suggestion
interface JsonLogEvent {
    timestamp: Dayjs,
    level: LOG_LEVEL,
    fields: JsonObject
}

const INVALID_TIMESTAMP_VALUE = 0;
```